ansible_collections/arista/avd/roles/eos_cli_config_gen/templates/eos/influxdb.j2

```diff
@@ -0,0 +1,51 @@
+{#
+ Copyright (c) 2023-2024 Arista Networks, Inc.
+ Use of this source code is governed by the Apache License 2.0
+ that can be found in the LICENSE file.
+#}
+{# eos - influxdb #}
+{% if influxdb is arista.avd.defined %}
+!
+monitor telemetry influx
+{%     if influxdb.destinations is arista.avd.defined %}
```

Comment:
This check is not needed, since it is included in the natural_sort below.

Reply:
Fixed.